idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" demote-non-dropping-particle="never" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Id&#x151;j&#xE1;r&#xE1;s - Quarterly Journal of the Hungarian Meteorological Service</title>
+    <title-short>Id&#x151;j&#xE1;r&#xE1;s</title-short>
+    <id>http://www.zotero.org/styles/idojaras-quarterly-journal-of-the-hungarian-meteorological-service</id>
+    <link rel="self" href="http://www.zotero.org/styles/idojaras-quarterly-journal-of-the-hungarian-meteorological-service"/>
+	<link href="http://www.met.hu/downloads.php?fn=/metadmin/newspaper/2016/03/b849ea3fc271ed1ff1bb69ca01063a9a-borito-3.pdf" rel="documentation"/>
+    <author>
+      <name>Adam Leelossy</name>
+      <email>leeladam@elte.hu</email>
+    </author>
+    <author>
+      <name>Eszter L. Toth</name>
+      <email>toth.eszter@itk.ppke.hu</email>
+    </author>
+    <issn>0324-6329</issn>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```

Comment:
we'd want something other than generic-base here. Probably "science" is the best fit from the available fields?
http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories